problem_builder/templates/html/dashboard.html

```diff
@@ -42,16 +42,30 @@
           {% for mcq in block.mcqs %}
             <tr>
               <th class="desc">{{ mcq.display_name }}</th>
-              <td class="value" {% if mcq.color %}style="border-right-color: {{mcq.color}};"{% endif %}>
-                {% if mcq.value %}{{ mcq.value }}{% endif %}
+              <td class="value"
+                  {% if mcq.color %} style="border-right-color: {{mcq.color}};"{% endif %}
+                  {% if not show_numbers %}
+                    {% if mcq.value %} aria-label="Score: {{mcq.value}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
+                  {% endif %}
+              >
+                {% if mcq.value and show_numbers %}
+                  {{ mcq.value }}
+                {% endif %}
               </td>
             </tr>
           {% endfor %}
           {% if block.has_average %}
            <tr class="avg-row">
-              <th class="desc">{% trans "Average" %}</th>
-              <td class="value" {% if block.average_color %}style="border-right-color: {{block.average_color}};"{% endif %}>
-                {{ block.average|floatformat }}
+              <th class="desc">{{ average_label }}</th>
+              <td class="value"
+                  {% if block.average_color %} style="border-right-color: {{block.average_color}};"{% endif %}
+                  {% if not show_numbers %}
+                    {% if block.average %} aria-label="Score: {{block.average|floatformat}}" {% else %} aria-label="{% trans 'No value yet' %}" {%endif%}
```

Comment:
@e-kolpakov @antoviaque this is the only a11y issue found in Mark's review. Can you guys submit a PR to fix it asap? Thanks!